driver-scala/src/main/scala/org/mongodb/scala/ChangeStreamObservable.scala

```diff
@@ -50,6 +50,18 @@ case class ChangeStreamObservable[TResult](private val wrapped: ChangeStreamPubl
     this
   }
 
+  /**
+   * Sets the fullDocument before change value.
+   *
+   * @param fullDocumentBeforeChange the fullDocument before change
```

Comment:
`ChangeStreamPublisher` documents its counterpart of this method with the words  _"Sets the fullDocumentBeforeChange value." There we can see that `fullDocumentBeforeChange` is a term. In `ChangeStreamObservable` the wording is _"Sets the fullDocument before change value."_, and is confusing, at least to me. I propose to use the same wording as that used in `ChangeStreamPublisher`.

Reply:
Fixed.  Done.